src/rules/noInternalModuleRule.ts

```diff
@@ -44,7 +44,7 @@ class NoInternalModuleWalker extends Lint.RuleWalker {
     }
 }
 
-function isNestedDeclaration(node: ts.ModuleDeclaration) {
+export function isNestedDeclaration(node: ts.ModuleDeclaration) {
```

Comment:
Can you instead move this function to `src/language/utils` and rename it `isNestedModuleDeclaration`? Then we avoid any chance of circular dependencies and keep shared code in a separate location